smoke/config.go

```diff
@@ -217,15 +226,18 @@ func validateIsolationSegments(config *Config) {
 	if !config.EnableIsolationSegmentTests {
 		return
 	}
-	if config.GetBackend() != "diego" {
+	if config.Backend != "diego" {
```

Comment:
Also, we prefer the accessor here over looking at the struct directly.  Could you please switch back to using `GetBackend()`, `GetIsolationSegmentName()` and `GetIsolationSegmentDomain()` instead?